lib/views/verify_code/verifiying_code_page.dart

```diff
@@ -19,7 +19,7 @@ class VerifyingCodePage extends StatelessWidget {
       backInkAction: () => {
         context.read<AuthModel>().accessToken = null,
         context.read<AuthModel>().user = null,
-        CommonNavigator.go(context, CommonNavigator.login)
+        CommonNavigator.go(context, CommonNavigator.sign, extra: {"register": false})
```

Comment:
Can't we do a back with GoRouter ?

Reply:
@taorepoara It seems that this route is at the root `localhost:10000/validation-user` and not under the `/sign` route. This means that either I have to move this route under the sign route or it should stay like this. What is your opinion on that ?